omigami/spec2vec.py

```diff
@@ -20,14 +20,17 @@ class InvalidCredentials(Exception):
 
 
 class Spec2Vec:
-    _endpoint_url = (
+    _PREDICT_ENDPOINT_POSITIVE_ION_MODE = (
```

Comment:
why did you upper case this one? it's not a global variable or used anywhere else

Reply:
I like to indicate constants like that too